pyblazing/pyblazing/apiv2/context.py

```diff
@@ -176,32 +183,22 @@ def collectPartitionsRunQuery(
         accessToken,
         config_options,
         single_gpu=False):
+
     import dask.distributed
-    worker_id = dask.distributed.get_worker().name
+    worker = dask.distributed.get_worker()
     for table_name in tables:
         if(isinstance(tables[table_name].input, dask_cudf.core.DataFrame)):
             if single_gpu:
```

Comment:
@felipeblazing please double check to make sure this `single_gpu` use case looks correct